monarchmoney/monarchmoney.py

```diff
@@ -33,6 +34,10 @@ def getLoginEndpoint(cls) -> str:
     def getGraphQL(cls) -> str:
         return cls.BASE_URL + "/graphql"
 
+    @classmethod
+    def getAccountBalanceHistoryUploadEndpoint(cls):
```

Comment:
nit: define the return type of this function

(sorry I missed this earlier)

Reply:
No worries - added!